src/convertToObject.js

```diff
@@ -6,7 +6,35 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // Transform string into array of lines
+  const propertiesArray = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There is no need to make a copy of the array before performing operations like `split`. The `split` method does not mutate the original string, so you can chain other methods directly to it.